### PR TITLE
Update identify-a-hold-on-an-exchange-online-mailbox.md

### PR DESCRIPTION
**Pull request by technicalkiwi**
Add in Identity Parameter to Set-RetentionCompliancePolicy Command under Next Steps

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/identify-a-hold-on-an-exchange-online-mailbox.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ Keep the following things in mind when managing a mailbox on delay hold:
 
 After you identify the holds that are applied to a mailbox, you can perform tasks such as changing the duration of the hold, temporarily or permanently removing the hold, or excluding an inactive mailbox from a Microsoft 365 retention policy. For more information about performing tasks related to holds, see the one of the following topics:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/identify-a-hold-on-an-exchange-online-mailbox.md]:
```suggestion
After you identify the holds that are applied to a mailbox, you can perform tasks such as changing the duration of the hold, temporarily or permanently removing the hold, or excluding an inactive mailbox from a Microsoft 365 retention policy. For more information about performing tasks related to holds, see one of the following topics:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - although not related to this PR, please also correct the typo in Line 233 as indicated.

**Comment by yogkumgit** [comment]
@technicalkiwi please implement the review comments. Thanks.

**Pull request merged**